src/common/locallaplaciancl.c

```diff
@@ -130,58 +130,40 @@ cl_int dt_local_laplacian_cl(
 
   if(b->bwidth <= 1 || b->bheight <= 1) return err;
 
-  size_t sizes_pad[] = { ROUNDUPDWD(b->bwidth, b->devid), ROUNDUPDHT(b->bheight, b->devid), 1 };
```

Comment:
Seems like the issue comes from both using sizes_pad. If I revert this chunk and the other one below I have no more errors.